### ISSUE TITLE
I am no longer able to post to my blogger accounts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#819]
```
What's the problem?

I have sent 3 requests asking why about 2 months ago scribe fire will not 
publish to my blogger accounts. I have reinstall the latest version of 
scribefire and still nothing. 

I have deleted all the accounts and still it won't work, and i have use brand 
new blogger accounts and newly created google accounts and it is not working. 

Can you resolve this on your end?

Jay
bulkbits@gmail.com



What browser are you using?

chrome



What version of ScribeFire are you running? 4.1 or what ever the latest version 
is... 

```

-----
Original issue reported on code.google.com by `bulkb...@gmail.com` on 14 Feb 2014 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
please email me what i need to do to fix this issue..

bulkbits@gmail.com
```

-----
Original comment by `bulkb...@gmail.com` on 14 Feb 2014 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:58
* Changed state: **Fixed**
